rails/benchmarks/bm_scaffold_create.rb

```diff
@@ -4,13 +4,6 @@
 require 'active_record'
 require_relative 'support/benchmark_rails'
 
-ENV['RAILS_ENV'] = 'production'
-
-class NullLogger < Logger
-  def initialize(*args);end
-  def add(*args, &block);end
-end
```

Comment:
@kirs I think it should be fine since we run the benchmarks in containers that we throw away after each run. :smile_cat: